sklearn/kernel_approximation.py

```diff
@@ -27,6 +26,7 @@
 from .utils._param_validation import Interval
 from .utils.extmath import safe_sparse_dot
 from .utils.validation import check_is_fitted
+from .utils._param_validation import Interval
```

Comment:
Another instance of a reimport that is not needed. I believe this is the last one.